### PR TITLE
Add seaside icons

### PR DESCRIPTION
**Pull request by danielbayley**
`waves-birds` can indicate a sea view e.g for accommodation on a travel site…

`waves-shark-fin` uses the same shape as `eclipse` in https://github.com/lucide-icons/lucide/pull/1280#issuecomment-1549433462, which was also taken directly from the centre of the existing [`sun-moon`](https://lucide.dev/icons/sun-moon).

Related to #1479 and #1488.

### CODE REVIEW

File: icons/bucket.svg
Comment (karsa-mistmere) [inline icons/bucket.svg:1]: Not sure about this one, I had to look up what it was to even recognise it.

File: icons/crab.svg
Comment (karsa-mistmere) [inline icons/crab.svg:1]: Cute crab, although I feel like at 100% the areas highlighted in red feel quite dense.
image

Comment (danielbayley) [reply icons/crab.svg:1]: > at 100% the areas highlighted in red appear quite dense.

@karsa-mistmere How about now (see latest `push`…)

File: icons/bucket.svg
Comment (danielbayley) [reply icons/bucket.svg:1]: You don’t think that looks like a bucket?! 😅

File: icons/crab.svg
Comment (karsa-mistmere) [reply icons/crab.svg:1]: To be honest, I think it looked better before. 😅 

I thought maybe the claws could be decreased in size / changed in shape, but maybe your original design was best all along:
image

Comment (danielbayley) [reply icons/crab.svg:1]: @karsa-mistmere I like the last one actually. I couldn’t fit the eyes in before…

**Comment by JClackett** [comment]
Was working on a surfboard icon so thought I'd share here as another option

Screenshot 2023-07-23 at 10 37 52

File: icons/bucket.svg
Comment (karsa-mistmere) [reply icons/bucket.svg:1]: Yeah, I didn't think it did and I still don't quite. 😅 
It's also a bit too small. Some alternate design ideas:
image

https://www.figma.com/file/1aIxerXRbbetwf3rFaRipi/lucide%2Fbucket?type=design&node-id=0%3A1&mode=design&t=EqIJnwxOMp6aPOqL-1

File: icons/surfboard.svg
Comment (karsa-mistmere) [inline icons/surfboard.svg:1]: The bottom left part is far too dense, arguably it's even a guideline violation.

File: icons/waves-birds.svg
Comment (karsa-mistmere) [inline icons/waves-birds.svg:1]: This one's a bit too large, maybe let's try an alt with just two wavelines.

File: icons/surfboard.svg
Comment (danielbayley) [reply icons/surfboard.svg:1]: @karsa-mistmere Fixed I think…

File: icons/bucket.svg
Comment (danielbayley) [reply icons/bucket.svg:1]: > Some alternate design ideas:

Nice, will see what works best…

Comment (danielbayley) [reply icons/bucket.svg:1]: @karsa-mistmere So I think it’s between these…

preview

Thinking we should have the flat 2D version that should match https://lucide.dev/icons/trash as much as possible, but then possibly along with a more 3D variant… Or just the one? What do you think @karsa-mistmere cc: @jguddas @ericfennis

**Comment by ericfennis** [comment]
@danielbayley Can you apply the new[ PR template](https://github.com/lucide-icons/lucide/blob/main/.github/pull_request_template.md) on this PR?

**Comment by ericfennis** [comment]
This PR is merged to the lab branch. This PR does not match our [PR template](https://github.com/lucide-icons/lucide/blob/main/.github/pull_request_template.md), which requires valid use cases. To move on we merged this PR to the [lab branch](https://github.com/lucide-icons/lucide/tree/lab), which will be moved to a separate repo in the future.

**Pull request merged**